src/GraphQLServiceProvider.php

```diff
@@ -20,8 +20,20 @@ public function register()
     {
         // Configure
         $this->app->configure('graphql');
+        // Load graphiql view
         $this->loadViewsFrom(dirname(__DIR__) . '/resources/views', 'graphql');
```

Comment:
If we want to support both Lumen and Laravel we need an if-block here.

Reply:
Like so: https://github.com/digiaonline/lumen-cors/blob/master/src/CorsServiceProvider.php#L15

Reply:
I think we should focus mostly on Lumen for now

Reply:
This is a standard way of adding support for both frameworks. I don't understand why we would do it differently in this project?